MNI_ecosystem.html

```diff
@@ -35,45 +35,14 @@
     <div class="slides">
       
      	<section>
-	  
-	  <!-- FIRST OPTION FIRST PAGE -->
-	  <section>
-
-	    <h1>THE MNI Ecosystem</h1>
-	    
-	    <h4 style="padding-bottom:0px; margin-bottom:0px;" align="center"><font color="f27c30">Database management and processing ecosystem for data-sharing initiatives</font></h4>
-	
-	    <!--ENTER DATE OF PRESENTATION HERE-->
-
-            <p><small>NOVEMBER 2017</small></p>
-            <p><small>Samir Das<br>Director of Technology</small></p>
-	    <p><small><font color="12a3d8"><b>McGill Centre for Integrative Neuroscience</b></font><br><font color="12a3d8"><b>Montreal Neurological Institute</b></font></small></p> 
-
-	    <table class="reveal">
-              <tr>
-		<td width="40%" style="vertical-align: middle">
-		  <a href="https://mcin-cnim.ca/" target="_blank"><img src="images/MCIN.png" style="border: 0"></a>
-		</td>
-		<td width="15%" style="vertical-align: middle">
-		  <a href="http://www.mcgill.ca/neuro/" target="_blank"><img src="images/Neuro.png" style="border: 0"></a>
-		</td>
-		<td width="35%" style="vertical-align: middle">
-		  <a href="http://ludmercentre.ca/" target="_blank"><img src="images/ludmer-new-white.png" style="border: 0"></a>
-		</td>
-	      </tr>
-	    </table>
-
-	  </section>
 
-	  <!-- SECOND OPTION FIRST PAGE -->
+	  <!-- FIRST PAGE -->
```

Comment:
below: 
Database management and processing -> **Data** management and processing

Reply:
done